Makefile

```diff
@@ -112,6 +112,11 @@ install: generate cert-mgr ## Install rukpak
 
 run: build-container kind-cluster kind-load install ## Build image, stop/start a local kind cluster, and run operator in that cluster
 
+wait:
+	kubectl wait --for=condition=Available --namespace=$(RUKPAK_NAMESPACE) deployment/plain-provisioner --timeout=60s
+	kubectl wait --for=condition=Available --namespace=$(RUKPAK_NAMESPACE) deployment/rukpak-core-webhook --timeout=60s
+	kubectl wait --for=condition=Available --namespace=crdvalidator-system deployment/crd-validation-webhook --timeout=60s
```

Comment:
This was added in #392 so we just need to rebase this PR to pick up this new Makefile target.

Reply:
Rebased in.